pkgs/top-level/all-packages.nix

```diff
@@ -25821,6 +25821,8 @@ with pkgs;
   freeradius = callPackage ../servers/freeradius { };
 
   freshrss = callPackage ../servers/web-apps/freshrss { };
+  freshrss-utils = callPackage ../servers/web-apps/freshrss/extensions/freshrss-utils.nix { };
```

Comment:
I think this line can be excluded

Reply:
Ok my idea here: make the `freshrss-utils` available, so users could provide extensions themself: https://github.com/NixOS/nixpkgs/blob/d33c545c3570fa544887fe1ae3d85bfd75ff6e00/nixos/modules/services/web-apps/freshrss.nix#L34